### PR TITLE
bug: Timezone of Event Time/Date and Purchase Time/Date changed to ev…

### LINKED ISSUE #8248: Ticket Email: Timezone of Event Time/Date and Purchase Time/Date should be following event Timezone
The organizer and attendee ticket email show the time and date in UTC instead of the event timezone.

Expected: Always show the emails in the event date and time following the event timezone.

- Currently the Purchase Date and Time are also wrong. They do not actually show the correct UTC time.
- The Purchase Date and Time are should follow the 24 hour format same as the event date/time.

Screenshot from 2021-09-28 14-51-00

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #8248 

**Ticket purchased organizer mail**:
ss_1632987934

**Attendee ticket**:
ss_1632985941

### CODE REVIEW

**Review by mariobehling — APPROVED** [review]

File: app/api/helpers/mail.py [hunk 1]
```diff
@@ -594,9 +597,11 @@ def send_order_purchase_organizer_email(order, recipients):
         site_link=order.event.site_link,
         order_url=order.site_view_link,
         event_date=order.event.starts_at.strftime('%d %B %Y'),
-        event_time=order.event.starts_at.strftime("%H:%M %Z"),
-        purchase_time=order.completed_at,
+        event_time=order.event.starts_at.replace(tzinfo=pytz.timezone('UTC'))
+        .astimezone(pytz.timezone(order.event.timezone))
```
Comment (maze-runnar) [inline app/api/helpers/mail.py:601]: this will change the time to event timezone or local timezone?

Comment (progmatic-99) [reply app/api/helpers/mail.py:601]: to event timezone. what do you mean by local timezone??

Comment (maze-runnar) [reply app/api/helpers/mail.py:601]: I mean, the problem statement is to make it timezone aware.

Comment (progmatic-99) [reply app/api/helpers/mail.py:601]: yes, change it to the timezone of the event.

Comment (maze-runnar) [reply app/api/helpers/mail.py:601]: @mariobehling can clear your doubt here

Comment (progmatic-99) [reply app/api/helpers/mail.py:601]: I don't have any doubt here.

Comment (mariobehling) [reply app/api/helpers/mail.py:601]: Time should be in event timezone.

Comment (maze-runnar) [reply app/api/helpers/mail.py:601]: In this case, changes in filters.py are not required, they will affect other areas. So please only make changes in helpers/mail.py @progmatic-99

Comment (progmatic-99) [reply app/api/helpers/mail.py:601]: In other areas too, event timezone is not shown. If I change back the filter, we would encounter the bug again.

**Pull request merged**